tex/i/_0054_0055_0056_0057_0058.tex

```diff
@@ -111,6 +112,7 @@
 маштабу становлять і первісні назви грошового маштабу,
 або маштабу цін.
 
+\looseness=1
```

Comment:
Can you try \loseness=-1 here?

Reply:
Yes, it can be done but the footnote 60 will either break or there will be extra blank like at the end of last para on page 64. That is why I extended this to prevent the weird spacing and preventing footnote break on page 64 and page 65.